ci_support/run_docker_build.sh

```diff
@@ -41,10 +58,19 @@ conda clean --lock
 conda install --yes --quiet conda-forge-build-setup
 source run_conda_forge_build_setup
 
-# Embarking on 1 case(s).
-    set -x
-    export CONDA_PERL=5.20.3.1
-    set +x
-    conda build /recipe_root --quiet || exit 1
-    upload_or_check_non_existence /recipe_root conda-forge --channel=main || exit 1
+set +e
+echo "[REMOVE] calling conda build on recipe_root"
+conda build /recipe_root --debug --quiet || echo "[REMOVE] conda build rv: \$?"
```

Comment:
This last "[REMOVE] ..." message never shows up on the screen.

Running conda build on on --debug mode also does not show anything right away.

I am really out of ideas here, any help is appreciated.

cc: @isuruf, @jakirkham